packages/@aws-cdk/aws-iam/lib/user.ts

```diff
@@ -159,16 +162,18 @@ export class User extends Resource implements IIdentity, IUser {
         return { statementAdded: true, policyDependable: this.defaultPolicy };
       }
 
-      public addToGroup(_group: IGroup): void {
-        throw new Error('Cannot add imported User to Group');
+      public addToGroup(group: IGroup): void {
+        this.groups.push(group.groupName);
       }
 
-      public attachInlinePolicy(_policy: Policy): void {
-        throw new Error('Cannot add inline policy to imported User');
+      public attachInlinePolicy(policy: Policy): void {
```

Comment:
Will this actually add an inline policy to an already-existing `User` object? I don't think so.